autogen/agentchat/conversable_agent.py

```diff
@@ -616,13 +616,13 @@ async def a_receive(
         if reply is not None:
             await self.a_send(reply, sender, silent=silent)
 
-    def _prepare_chat(self, recipient, clear_history):
+    def _prepare_chat(self, recipient: "ConversableAgent", clear_history: bool, prepare_recipient: bool = True) -> None:
```

Comment:
```suggestion
    def _prepare_chat(self, recipient: ConversableAgent, clear_history: bool, prepare_recipient: bool = True) -> None:
```
I think it's a class instead of a string, right?

Reply:
https://peps.python.org/pep-0484/#forward-references

But looks like there is actually a better way https://peps.python.org/pep-0563/, will try it out

Reply:
Thank you. I've learnt something new. :)